recipes/openmc-plotter/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "openmc-plotter" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openmc-dev/plotter/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: f54644486eed1b87ae4959a464bbce0a55f625508ac99afd2b5606c7312811e8
+
+build:
+  entry_points:
+    - openmc-plotter=openmc_plotter.__main__:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - matplotlib-base
+    - numpy
+    - openmc >0.12.2
+    - pyside2
+    - python  >=3.6
+    - ipython
```

Comment:
I cannot see this upstream?

Reply:
This comes from somewhere in `openmc`, which [doesn't](https://github.com/conda-forge/openmc-feedstock/blob/926cb818dc80533c95d0c46f0fc4261efe784179/recipe/meta.yaml#L102-L110) run `pip check`

Reply:
You can make a PR there and add it here until it is merged. Just add a link to the PR to the meta.yaml here as a note.

Reply:
Thanks for making that PR @ndevenish - I will merge that and then I think this would be ready to go after that trickles through.

Reply:
Oh, haha, I didn't clock that it was you for both repos - NP

Reply:
> If there are issues merging, then it's probably okay to either a) include ipython, or b) not run pip check - I don't know what the staged-recipes maintainers preference would be on that.

I always prefer to have a working recipe so if something else is broken, make a PR to fix it there and add it here until the other PR is merged. Better to add something than to ship something that's broken. :)